RubberduckTests/QuickFixes/IntroduceLocalVariableForRedimQuickFixTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using NUnit.Framework;
+using Rubberduck.Inspections.Concrete;
+using Rubberduck.Inspections.QuickFixes;
+using RubberduckTests.Inspections;
+using RubberduckTests.Mocks;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace RubberduckTests.QuickFixes
+{
+    [TestFixture]
+    public class IntroduceLocalVariableForRedimQuickFixTests
+    {
+        [Test]
+        [Category("QuickFixes")]
+        public void VariantRedimStatement_QuickFixWorks()
```

Comment:
I think we need another unit test to verify that quick fix will only fix the first occurrence of the `Redim` on same variable and on an unrelated variable.

e.g. 
```
Redim foo(1)
Redim foo(2)
```

=>

```
Dim foo() As Variant
Redim foo(1)
Redim foo(2)
```

Likewise, we should have a test for `Redim foo(1) As Long` => `Dim foo() As Long` & `Redim foo(1)` and also the special case of 

```
Redim foo(1) As Long
Redim foo(1) As Long
```

in which we need to remove the `As Long` for both.